src/components/legalContent/legalTextReader.styles.scss

```
@@ -0,0 +1,256 @@
+		justify-content: center;
+		padding: 0;
+		border: 0;
+		border-radius: 50%;
+		background: transparent;
+		color: var(--m3-on-surface-variant, #444748);
+		cursor: pointer;
+
+		svg {
+			width: 20px;
+			height: 20px;
+			fill: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)